src/Orleans.Core/Configuration/Options/GrainCollectionOptions.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+
+namespace Orleans.Hosting
+{
+    /// <summary>
+    /// Silo options for grain garbage collection.
+    /// </summary>
+    public class GrainCollectionOptions
```

Comment:
I think this class should be in Orleans.Runtime, since the component it configures lives there.